src/System/Wlog/Launcher.hs

```diff
@@ -124,3 +127,42 @@ buildAndSetupYamlLogging configBuilder loggerConfigPath = do
 -- See this module description.
 initLoggingFromYaml :: MonadIO m => FilePath -> m ()
 initLoggingFromYaml = buildAndSetupYamlLogging mempty
+
+-- | Initializes logging using given 'FilePath' to logger configurations,
+-- runs the action with the given 'LoggerName'.
+runLoggingWithFile :: MonadIO m => FilePath -> LoggerName -> LoggerNameBox m a -> m a
+runLoggingWithFile filename logName f = do
+    buildAndSetupYamlLogging productionB filename
+    usingLoggerName logName f
+
+-- | Default logging configuration with the "node" logger.
```

Comment:
I would like to see in documentation what is enabled by default in default logging. Like, time is shown, severity for root logger, severity for given logger, etc. You don't need to describe things like _logger rotation is disabled_. Describe only differences from `mempty :: LoggerConfig`.